### PR TITLE
fix: evn `_MS` to String env

### PR DESCRIPTION
**Pull request by jiho-kr**
env `xx_ms` is deprecated.

Provides env in the form of a string.

There is no breaking change when using the old env.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,48 +95,52 @@ $ RABBITMQ_HOST=localhost npm t
...
+| ISLAND_RPC_RES_NOACK                    | boolean | false                     |                                                                   |                                                                |
+| ISLAND_NO_REVIVER                       | boolean | false                     |                                                                   | 'NO_REVIVER'                                                   |
+| ISLAND_USE_REVIVER                      | boolean | false                     |                                                                   | 'USE_REVIVER'                                                  |
+| ISLAND_STATUS_EXPORT                    | boolean | false                     | If it is true, use island-status-exporter                         | 'STATUS_EXPORT'                                                |
+| ISLAND_STATUS_EXPORT_TIME_MS            | number  | 10 * 1000                 | Time to save file for instance status                             | 'STATUS_EXPORT_TIME_MS'                                        |
```
Comment (heycalmdown) [inline README.md]: Don't leave this variable behind.

File: README.md [hunk 2]
```diff
@@ -95,48 +95,52 @@ $ RABBITMQ_HOST=localhost npm t
...
+| ISLAND_REDIS_PORT                       | number  | 6379                      |                                                                   | 'REDIS_PORT'                                                   |
+| ISLAND_MONGO_HOST                       | string  | 'mongodb://mongodb:27017' |                                                                   | 'MONGO_HOST'                                                   |
+| ISLAND_RPC_DISTRIB_SIZE                 | number  | 16                        |                                                                   |                                                                |
+| ISLAND_USE_CIRCUIT_BREAK                | boolean | false                     |                                                                   |                                                                |
+| ISLAND_CIRCUIT_BREAK_TIME_MS            | number  | 60 * 1000                 |                                                                   |                                                                |
```
Comment (heycalmdown) [inline README.md]: this too

File: README.md [hunk 3]
```diff
@@ -95,48 +95,52 @@ $ RABBITMQ_HOST=localhost npm t
...
+| ISLAND_CIRCUIT_BREAK_FAILRATE_THRESHOLD | number  | 0.2                       |                                                                   |                                                                |
+| ISLAND_CIRCUIT_BREAK_REQUEST_THRESHOLD  | number  | 10                        |                                                                   |                                                                |
+| ISLAND_FLOWMODE_DELAY                   | number  | 10000                     |                                                                   |                                                                |
```
Comment (heycalmdown) [inline README.md]: this also

Comment (jiho-kr) [reply README.md]: ok got it

File: README.md (hunk 2 above)
Comment (jiho-kr) [reply README.md]: ok got it

File: README.md (hunk 1 above)
Comment (jiho-kr) [reply README.md]: ok got it

**Pull request merged**